charts/linkerd-control-plane/values.yaml

```diff
@@ -455,3 +455,8 @@ nodeSelector:
 # [K8S documentation](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/)
 # for more information
 #tolerations:
+
+# -|- NodeAffinity section, See the
+# [K8S documentation](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity)
+# for more information
+#nodeAffinity:
```

Comment:
This is not rendered into `README.md` (similiar to `tolerations`). Moreover, we are lacking `enablePodAntiAffinity` description here. Is it a good idea to add it all?

Reply:
You can have those docs added automatically into the README by uncommenting them, and leaving their values empty, but you'll have to verify that doesn't break the rendering of the manifests. I'd be nice to add docs for `enablePodAntiAffinity`, but I believe the docs from `values-ha.yaml` aren't picked up in the README.